packages/pluggableWidgets/datagrid-web/tests/e2e/specs/DataGrid.spec.ts

```diff
@@ -0,0 +1,93 @@
+import page from "../../../../../../configs/e2e/src/pages/page";
+
+describe("datagrid-web", () => {
+    beforeAll(() => {
+        page.open(); // resets page
+    });
+
+    describe("capabilities: sorting", () => {
+        it("changes order of data to ASC when clicking sort option", () => {
+            const datagrid = page.getWidget("datagrid1");
+            datagrid.waitForExist();
+            const column = page.getElement(".column-header*=First Name", datagrid);
+            const icon = page.getElement("svg", column);
+            column.click();
+            expect(icon.getAttribute("data-icon")).toContain("long-arrow-alt-up");
+        });
+        it("changes order of data to DESC when clicking sort option", () => {
+            const datagrid = page.getWidget("datagrid1");
+            const column = page.getElement(".column-header*=First Name", datagrid);
+            const icon = page.getElement("svg", column);
+            datagrid.waitForExist();
+            column.click();
+            expect(icon.getAttribute("data-icon")).toContain("long-arrow-alt-down");
+        });
+    });
+    describe("capabilities: filtering", () => {
+        it("changes the result showed in the data grid following the filter inserted", () => {});
+    });
+    describe("capabilities: resizing", () => {
+        it("changes the size of the column", () => {
+            const datagrid = page.getWidget("datagrid1");
+            const column = page.getElement(".th*=Age", datagrid);
+            const resizer = page.getElement(".column-resizer", column);
+            datagrid.waitForExist();
+            const initialSize = Math.round(column.getSize("width"));
+            browser.performActions([
+                {
+                    type: "pointer",
+                    id: "pointer1",
+                    parameters: { pointerType: "mouse" },
+                    actions: [
+                        { type: "pointerMove", origin: resizer, x: 0, y: 0 },
+                        { type: "pointerDown", button: 0 },
+                        { type: "pause", duration: 10 },
+                        { type: "pointerMove", origin: resizer, x: 30, y: 0 },
+                        { type: "pointerUp", button: 0 },
+                        { type: "pause", duration: 10 }
+                    ]
+                }
+            ]);
+            const finalSize = Math.round(column.getSize("width"));
+            expect(finalSize).toBe(initialSize + 30);
+        });
+    });
+    describe("capabilities: reordering", () => {
+        // eslint-disable-next-line jest/no-disabled-tests
+        xit("changes the order of the columns", () => {
+            const datagrid = page.getWidget("datagrid1");
+            const column = page.getElement(".column-header*=Age", datagrid);
+            // const column2 = page.getElement(".column-header*=First", datagrid);
```

Comment:
This should be removed if it's not relevant.

Reply:
Done!